internal/bundle/bundle.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package bundle
```

Comment:
What is this package for? I see some logic for directories and as well as OCI stuff. Can we add a `doc.go` file describing the point of this package?

Reply:
It's just some helper utilities for interacting with a Bundle dir. Can add doc.go though, sure